pkg/exporter/process.go

```diff
@@ -42,35 +42,71 @@ type ExportingProcess struct {
 	connToCollector net.Conn
 	obsDomainID     uint32
 	seqNumber       uint32
+	templateID      uint16
 	set             *entities.Set
 	msg             *entities.MsgBuffer
 	templatesMap    map[uint16]templateValue
+	templateRefCh   chan struct{}
 }
 
-func InitExportingProcess(collectorAddr net.Addr, obsID uint32) (*ExportingProcess, error) {
+// InitExportingProcess takes in collector address(net.Addr format), obsID(observation ID) and tempRefTimeout
```

Comment:
Maybe we can specify the tempRefTimeout unit (seconds) here

Reply:
done